### PR TITLE
Campaign goal prop

### PR DESCRIPTION
**Pull request by noweverywhere**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "author": "EverydayHero <edh-dev@everydayhero.com>",
   "name": "edh-widgets",
-  "version": "0.2.5",
+  "version": "0.2.6",
```
Comment (bjbrewster) [inline package.json]: Can we bump the revision in a separate PR please.  There's a couple of things to be released.

Comment (noweverywhere) [inline package.json]: We will do that by the end of the day? I would like to have my story finished before demo tomorrow.

File: src/components/totals/GoalProgress/index.js [hunk 2]
```diff
@@ -49,6 +51,8 @@ module.exports = React.createClass({
     var offsetWidth = numeral(progress).format('0%');
     var style = { width: offsetWidth || '100%' };
 
+    if (!progress) { return null; }
```
Comment (bjbrewster) [inline src/components/totals/GoalProgress/index.js]: You don't have to return anything here.  To be more consise, you could just update the return statement below to `return progress && (`

File: src/components/totals/GoalProgress/index.js [hunk 3]
```diff
@@ -74,13 +78,25 @@ module.exports = React.createClass({
 
   renderText: function() {
     var props = this.props;
+    var message;
+
+    if(!props.goal && props.fallbackToFundsRaised) {
+
```
Comment (bjbrewster) [inline src/components/totals/GoalProgress/index.js]: Can you leave out the empty lines at beginning and end of if/else blocks please?

File: src/components/totals/GoalProgress/index.js [hunk 4]
```diff
@@ -74,13 +78,25 @@ module.exports = React.createClass({
 
   renderText: function() {
     var props = this.props;
+    var message;
+
+    if(!props.goal && props.fallbackToFundsRaised) {
```
Comment (bjbrewster) [inline src/components/totals/GoalProgress/index.js]: Can this block be extracted into separate function to simplify the renderText function:

```
  getMessage: function() {
    if(!props.goal && props.fallbackToFundsRaised) {
      return this.tm('no_goal_text', {
        total: this.formatCurrency(props.total)
      });
    } else {
      ...
    }
  },
```

File: src/components/totals/GoalProgress/index.js [hunk 5]
```diff
@@ -74,13 +78,25 @@ module.exports = React.createClass({
...
+      });
+    }
 
     return (
       <div className="GoalProgress__text">
-        { this.tm('text', {
-          total: this.formatCurrency(props.total),
-          goal: this.formatCurrency(props.goal)
-        }) }
+        { message }
```
Comment (bjbrewster) [inline src/components/totals/GoalProgress/index.js]: { this.getMessage() }

**Comment by bjbrewster** [comment]
:+1:

**Pull request merged**